### PR TITLE
vue: docs updated for range-input component

### PR DESCRIPTION
**Pull request by anik-ghosh-au7**
Description - `RangeInput` component added in vue-searchbox

please look into this PR for further reference - https://github.com/appbaseio/reactivesearch/pull/1690

### CODE REVIEW

File: content/docs/reactivesearch/vue/range/RangeInput.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    show the selected item as a filter in the selected filters view. Defaults to `true`.
+-   **snap** `Boolean` [optional]
+    makes the slider snap on to points depending on the `stepValue` when the slider is released. Defaults to `true`. When set to `false`, `stepValue` is ignored.
+-   **stepValue** `Number` [optional]
+    step value specifies the slider stepper. Value should be an integer greater than or equal to `1` and less than `Math.floor((range.end - range.start) / 2)`. Defaults to 1.
+-   **URLParams** `Boolean` [optional]
+    enable creating a URL query string parameter based on the selected value of the list. This is useful for sharing URLs with the component state. Defaults to `false`.
+-   **includeNullValues** `Boolean` [optional]
+    If you have sparse data or document or items not having the value in the specified field or mapping, then this prop enables you to show that data. Defaults to `false`.
+-   **componentStyle** `Object` [optional]
```
Comment (bietkul) [inline content/docs/reactivesearch/vue/range/RangeInput.md]: @anik-ghosh-au7 Remove it for now

File: content/docs/reactivesearch/vue/range/RangeInput.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    selects a initial range values using `start` and `end` key values from one of the data elements.
+-   **value** `Object` [optional]
+    controls the current value of the component.It selects the data from the range (on mount and on update). Use this prop in conjunction with `change` event.
+-   **validateRange** `function` [optional]
+    is a callback function that can be used to validate the range input values before applying it. This function accepts an array of numbers where first element represents the `start` range and second element represents the `end` range. The following example prevents the users to type negative value for start range input.
+
+```html
+<template>
+	<range-input
+		:validateRange="[start, end] => {
```
Comment (bietkul) [inline content/docs/reactivesearch/vue/range/RangeInput.md]: @anik-ghosh-au7 Do you think that if it is the correct syntax?

File: content/docs/reactivesearch/vue/range/RangeInput.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+</template>
+<script>
+	export default {
+		name: 'app',
+		methods: {
+			getCustomQuery: (value, props) => {
+				return {
+					query: {
+						match: {
+							data_field: 'this is a test',
```
Comment (bietkul) [inline content/docs/reactivesearch/vue/range/RangeInput.md]:
```suggestion
							price: 1000,
```

**Review by bietkul — CHANGES_REQUESTED** [review]

**Comment by bietkul** [comment]
@anik-ghosh-au7 Embed the codesandbox link. Add an example in Repo.

**Comment by anik-ghosh-au7** [comment]
@bietkul I have done the updates as you have mentioned

**Review by bietkul — APPROVED** [review]
lgtm 👍

**Pull request merged**